### DISCUSSION TITLE
Separate Binder environment per-page

### DISCUSSION THREAD

**Discussion by pgierz** [executablebooks/meta#775] [Q&A]
Hello,

I was wondering if it is possible to create unique launch-binder environments for each notebook I include, or they all must use the same? For some notebooks, the package list I need to install is rather extensive, for others, only minimal. Having a way to separate that would possibly cut down the spin-up time of the containers my binder does for you

Thanks!
Paul

**Comment by michaelweinold** [comment]
Compare my open issue https://github.com/executablebooks/jupyter-book/issues/1816 (still pending satisfactory resolutions).
